recipes/counterfactual_data_augmentation.ipynb

```diff
@@ -37,7 +37,7 @@
     "# !pip install kili # uncomment if you don't have kili installed already\n",
     "from kili.client import Kili\n",
     "\n",
-    "api_key = os.getenv('KILI_USER_API_KEY')\n",
+    "api_key = os.getenv('KILI_API_KEY')\n",
```

Comment:
this is the default behavior of the Kili client, so you can remove the `api_key` arg in all the Kili client initializations (here and elsewhere)